Builds/levelization/results/loops.txt

```diff
@@ -16,6 +16,9 @@ Loop: ripple.app ripple.overlay
 Loop: ripple.app ripple.peerfinder
   ripple.app > ripple.peerfinder
 
+Loop: ripple.app ripple.protocol
+  ripple.app > ripple.protocol
+
```

Comment:
This change indicates that a circular dependency was introduced between `src/ripple/app` and `src/ripple/protocol`. That's not great. It would be a lot better if the relevant places (which I haven't identified yet) were refactored so that nothing in `src/ripple/protocol` depended on anything in `src/ripple/app`.